doc/how_to/index.rst

```diff
@@ -7,3 +7,4 @@ How to guides
     get_started
     analyse_neuropixels
     handle_drift
+    load_matalb_data
```

Comment:
Note that you have a typo in the file name: 
```suggestion
    load_matlab_data
```

Reply:
Yep, thanks for the correction! Worse place to have a typo.